modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/groupby/BaseGroupByTestCase.java

```diff
@@ -509,16 +503,14 @@ protected void assertGroups(
 		Map<String, String> expectedCountsMap, Hits hits,
 		IndexingTestHelper indexingTestHelper) {
 
-		Map<String, Hits> hitsMap = hits.getGroupedHits();
-
-		Collection<Map.Entry<String, Hits>> entries = hitsMap.entrySet();
+		Map<String, String> actualCountsMap = new HashMap<>();
```

Comment:
@dantewang Hi Dante, I tried to follow this https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DMaps-are-plurals%2C-with-suffix

I just pushed SF rename on the files I touched in this PR. 
Is this understanding correct?
- expectedCounts and actualCounts are special names given based on what they do because its a Map<String, String>. We cannot name it StringsMap. Therefore, no Map is needed.
- hitsMap, fieldsMap, doublesMap are general names given based on the Map's type. So, Map should be added since hits, fields, doubles implies array or list of these types.

Reply:
@julschong Hmmmm, based on the link, it seems `expectedCountsMap` is also a valid name, because the name reflects what it does instead of what it contains. Let's see what Brian decides.

Reply:
@julschong I dropped the SF commit, let's see what Brian says.